packages/terra-graphs/src/components/Pie/PieChart.jsx

```diff
@@ -0,0 +1,42 @@
+/* eslint import/no-unresolved: off */
```

Comment:
Do we need this? It should be covered by one of the eslint rules: 
https://github.com/cerner/terra-graphs/blob/1ac036d8d82783bd0d00caa54e32f6ab57e33ca7/.eslintrc.yaml#L81

Reply:
Removed [here](https://github.com/cerner/terra-graphs/pull/138/commits/c73e2f34de761535cb328e09c1ccb73ee8b57b99#diff-bf3fa8a62a1df8ea368f4586284e7684871dafbbc568895e47fdbe55ed2ea8b3L1).